Week2/src/clients/budgetCheck.js

```diff
@@ -0,0 +1,31 @@
+import Budget from "../models/budget.js";
+
+import lockedSync from "locked-sync"
+const sync = lockedSync();
+
+
+export default async () => {
+
+    const end = await sync();
+    let budgetAmount = 0
+    try {
+        const userBudget = await Budget.find();
+        if (userBudget) {
+            budgetAmount = userBudget[0].amount
+        }
+        if (userBudget[0].amount > 0) {
+            console.log("Your current amount is ", userBudget[0].amount)
+            return true
+        }
+        else {
+            console.log("Your amount: ", userBudget[0].amount, "is not enough")
+            return false
+        }
+    } catch (err) {
+        console.log("Error trying to access user budget", err);
```

Comment:
Think a little bit of your response 🤔

Try to return always the same type ....

If everything is ok you are returning true or false but if there is a problem you are not returning anything (undefined), so maybe it is worth it to return false on the catch;